src/Executor/Executor.php

```diff
@@ -120,7 +120,7 @@ public static function execute(
      * Useful for async PHP platforms.
      *
      * @param mixed[]|null $rootValue
-     * @param mixed[]|null $contextValue
+     * @param mixed|null   $contextValue
```

Comment:
Could you also change `mixed[]` to `mixed` for the `$rootValue` as it has the same issue?

Reply:
done